bower.json

```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "bem-core": "2.5.1",
-    "bem-components": "git://github.com/bem/bem-components.git#v2.0.0",
+    "bem-core": "2.6.0",
+    "bem-components": "git://github.com/bem/bem-components.git#v2.1.0",
```

Comment:
@tadatuta some time ago there was some problem with it, but I forget which one :)
Now everything work correctly. Update PR.